src/sonic-ctrmgrd/tests/ctrmgr_iptables_test.py

```diff
@@ -103,30 +103,29 @@ def __init__(self, ret, stdout, stderr):
 
 
 def mock_subproc_run(cmd, shell, capture_output):
-    cmd_prefix = "sudo iptables -t nat "
-    list_cmd = "{}-n -L OUTPUT ".format(cmd_prefix)
-    del_cmd = "{}-D OUTPUT ".format(cmd_prefix)
-    ins_cmd = "{}-A OUTPUT -p tcp -d ".format(cmd_prefix)
+    list_cmd = ["sudo", "iptables", "-t", "nat", "-n", "-L", "OUTPUT"]
+    del_cmd = ["sudo", "iptables", "-t", "nat", "-D", "OUTPUT"]
+    ins_cmd = ["sudo", "iptables", "-t", "nat", "-A", "OUTPUT", "-p", "tcp", "-d"]
 
-    assert shell
+    assert shell == False
     
     print("cmd={}".format(cmd))
-    if cmd.startswith(list_cmd):
-        num = int(cmd[len(list_cmd):])
+    if list_cmd == cmd[:len(list_cmd)]:
+        num = int(cmd[-1])
```

Comment:
>-1 [](http://example.com/codeflow?start=22&length=2)

The logic is different from original logic, even I consider string vs. array of strings.

Reply:
Could you explain how so? 
I think the purpose is to get the `num` variable from this line, and UT tests in this repo pass.
https://github.com/sonic-net/sonic-buildimage/blob/8d7f66ab175aaca05395314a26b2b1dab85eb576/src/sonic-ctrmgrd/ctrmgr/ctrmgr_iptables.py#L108

Reply:
Maybe change to `int(cmd[len(list_cmd):][0])`?

Reply:
Suggest `int(cmd[len(list_cmd)])`, however need check index first.